src/declarative.js

```diff
@@ -107,7 +107,7 @@
     return moduleRecords[key] || (moduleRecords[key] = {
       key: key,
       dependencies: [],
-      module: new Module({}),
+      module: new Reflect.Module({}),
```

Comment:
See https://github.com/ModuleLoader/es6-module-loader/blob/0.50/src/loader.js#L682, we rely on internal scoping through the build process.

It may well be worth rewriting the build using ES modules and Rollup as another task here to make these scoping assumptions clearer!